docs/gpu-pci.md

```diff
@@ -0,0 +1,177 @@
+# GPU enabled clusters via PCI Passthrough
+
+## Overview
+
+Starting with the v1.3 release, Kubernetes clusters with nodes having GPU devices can be created on vSphere. You can choose to create a cluster with both worker and control plane nodes having GPU devices attached to them.
+
+Before we begin, a couple of important things of note:
+
+- [NVIDIA GPU Operator](https://github.com/NVIDIA/gpu-operator) is used to expose the GPU PCI devices to the workloads running on the cluster.
+- The OVA templates used for cluster creation should have the VMX version (Virtual Hardware) set to 17. This is necessary because Dynamic DirectPath I/O was introduced in this version, which enables the Assignable Hardware intelligent for passthrough devices.
```

Comment:
```suggestion
- The OVA templates used for cluster creation should have the VMX version (Virtual Hardware) set to 17. This is necessary because Dynamic DirectPath I/O was introduced in this version, which enables the Assignable Hardware intelligence for passthrough devices.
```